apps/web/screens/Organization/Syslumenn/Auctions.tsx

```diff
@@ -68,97 +398,96 @@ const Auctions: Screen<AuctionsProps> = ({ organizationPage, namespace }) => {
-      value: 'syslumadurinn-a-vesturlandi',
-    },
-    {
-      label: 'Vestfirðir',
-      value: 'syslumadurinn-a-vestfjordum',
-    },
-    {
-      label: 'Vestmannaeyjar',
-      value: 'syslumadurinn-i-vestmannaeyjum',
-    },
-    {
-      label: 'Suðurland',
-      value: 'syslumadurinn-a-sudurlandi',
-    },
-    {
-      label: 'Suðurnes',
-      value: 'syslumadurinn-a-sudurnesjum',
-    },
-    {
-      label: 'Höfuðborgarsvæðið',
-      value: 'syslumadurinn-a-hoefudborgarsvaedinu',
-    },
-  ]
-
-  const date = new Date()
+  const [officeLocation, setOfficeLocation] = useState<OfficeLocation>(
+    OFFICE_LOCATIONS[0],
+  )
+  const setOfficeLocationBySlugValue = (slugValue: string) => {
+    const targetOfficeLocationList = OFFICE_LOCATIONS.filter(
+      (o) => o.slugValue === slugValue,
+    )
+    const targetOfficeLocation =
+      targetOfficeLocationList.length > 0
+        ? targetOfficeLocationList[0]
+        : OFFICE_LOCATIONS[0] // Fallback to first Office Location
+    setOfficeLocation(targetOfficeLocation)
+  }
 
-  const months = [
-    {
-      label: 'Næstu uppboð',
-      value: '',
-    },
-  ]
+  const [query, _setQuery] = useState('')
+  const setQuery = (query: string) => _setQuery(query.toLowerCase())
 
-  const [organization, setOrganization] = useState<string>(
-    organizations[0].value,
+  const [lotTypeOption, setLotTypeOption] = useState<LotTypeOption>(
+    LOT_TYPES_OPTIONS[0],
   )
+  const setLotTypeOptionByValue = (value: string) => {
+    const targetLotTypeOptionList = LOT_TYPES_OPTIONS.filter(
+      (o) => o.value === value,
+    )
+    const targetLotTypeOption =
+      targetLotTypeOptionList.length > 0
+        ? targetLotTypeOptionList[0]
+        : LOT_TYPES_OPTIONS[0] // Fallback to first Lot Type Option
+    setLotTypeOption(targetLotTypeOption)
+  }
 
-  const [month, setMonth] = useState<string>(months[0].value)
+  const [date, setDate] = useState<Date>()
 
-  // Create options for the last three months in YYYY-MM format
-  for (let i = 0; i <= 2; i++) {
-    months.push({
-      label: format(date, 'MMMM yyyy'),
-      value: `${date.getFullYear()}-${date.getMonth()}`,
-    })
-    date.setMonth(date.getMonth() - 1)
-  }
+  const [showCount, setShowCount] = useState(10)
 
-  const { loading, error, data, refetch } = useQuery<
+  const { loading, error, data } = useQuery<
     Query,
-    QueryGetAuctionsArgs
-  >(GET_AUCTIONS_QUERY, {
+    QueryGetSyslumennAuctionsArgs
+  >(GET_SYSLUMENN_AUCTIONS_QUERY, {
     variables: {
-      input: {
-        lang: 'is',
-        organization,
-        ...(month && {
-          year: parseInt(month.split('-')[0]),
-          month: parseInt(month.split('-')[1]),
-        }),
-      },
+      input: {},
     },
   })
 
-  useEffect(() => {
-    refetch()
-  }, [organization, month, refetch])
+  // TODO: Fix input focus bug, where after after initial load of the page and first key-up on the Input, the Input looses focus.
```

Comment:
Raising this TODO in a comment.

We'll need to address this before merging this PR.